extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js

```diff
@@ -33,7 +33,8 @@ require([
 		videoTags = videoDetails.videoTags || '',
 		slotTargeting = {
 			plist: recommendedPlaylist,
-			vtags: videoTags
+			vtags: videoTags,
+			videoDedicatedFor: videoDetails.isDedicatedForArticle ? 'article' : 'wiki'
```

Comment:
What about `videoScope` instead of `videoDedicatedFor`?

Reply:
done in both app/mobile-wiki